ColorTypes.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <string>
+#include <map>
+
+namespace TelCoColorCoder
+{
+    enum MajorColor
+    {
+        WHITE,
+        RED,
+        BLACK,
+        YELLOW,
+        VIOLET
+    };
+    enum MinorColor
+    {
+        BLUE,
+        ORANGE,
+        GREEN,
+        BROWN,
+        SLATE
+    };
+
+    const char * const MajorColorNames[] = {
+        "White", "Red", "Black", "Yellow", "Violet"};
+
+    const unsigned numberOfMajorColors =
+        sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
+
+    const char * const MinorColorNames[] = {
+        "Blue", "Orange", "Green", "Brown", "Slate"};
+
+    const unsigned numberOfMinorColors =
+        sizeof(MinorColorNames) / sizeof(MinorColorNames[0]);
+
+    const std::map<MajorColor, const char*> MajorColorMap = {
+        {WHITE, "White"},
```

Comment:
Be careful about repeating string literals. E.g., if we need to release this in another language, we have to change all color strings in two places